releases/release-1.22/release-team.md

```diff
@@ -4,7 +4,7 @@
 |----------|----------------------------------|----------------------------------------|
 | Lead | Savitha Raghunathan ([@savitharaghunathan](https://github.com/savitharaghunathan) / Slack: `@sraghunathan`) | Anna Jung ([@annajung](https://github.com/annajung) / Slack: `@annajung`),Derrik Campau ([@erismaster](https://github.com/erismaster) / Slack: `@derrik campau`),Divya Mohan ([@divya-mohan0209](https://github.com/divya-mohan0209) / Slack: `@divya`), Kirsten Garrison ( [@kikisdeliveryservice](https://github.com/kikisdeliveryservice) / Slack: `@oikiki`) |
 | Enhancements | James Laverack ([@jameslaverack](https://github.com/jameslaverack) / Slack: `@james.laverack`)| Xander Grzywinski ([@salaxander](https://github.com/salaxander) / Slack: `@Xander`), Rey Lejano ([@reylejano](https://github.com/reylejano) / Slack: `@rlejano`), Grace Nguyen ([@gracenng](https://github.com/gracenng) / Slack: `@Grace Nguyen`), Supriya Premkumar ([@supriya-premkumar](https://github.com/supriya-premkumar) / Slack: `@supriya premkumar`), Joseph Sandoval ([@jrsapi](https://github.com/jrsapi) / Slack: `@Joseph`) |
-| CI Signal | Max Körbächer ([@mkorbi](https://github.com/mkorbi) / Slack: `@mkorbi`) | |
+| CI Signal | Max Körbächer ([@mkorbi](https://github.com/mkorbi) / Slack: `@mkorbi`) | [Sonia Singla](https://github.com/soniasingla) (@soniasingla / Slack: `@SONIA SINGLA`), [Hossein Salahi](https://github.com/encodeflush) (@encodeflush / Slack: `@hsalahi`), [Anna Kapuścińska](https://github.com/lambdanis) (@lambdanis / Slack: `@lambdanis`), [Rodolfo Martínez Vega](https://github.com/ramrodo) (@ramrodo / Slack: `@ramrodo`) |
```

Comment:
nit: I see the rest of the file links over the github usernames, i used the same, will be a bit weird to not have all the same. e.g.: Victor Palade ([@pi-victor](https://github.com/pi-victor)

Reply:
Can you open a follow-up fix for that? :)

Reply:
I made a mistake doing this too... maybe we should have a `release-team.yaml` file and generate this `.md` file with some Go?

Reply:
sure, i can open a PR to fix it. 

> I made a mistake doing this too... maybe we should have a `release-team.yaml` file and generate this `.md` file with some Go?

that doesn't sound like a bad idea, who's gonna do that?

Reply:
I could raise an issue for it?

Reply:
Sounds good @JamesLaverack. Let's discuss this on a dedicated issue.